tests/geospatial/test_landsat_analysis.py

```diff
@@ -0,0 +1,128 @@
+import collections
+import datetime
+import os
+
+import numpy as np
+import odc.stac
+import planetary_computer
+import pystac_client
+import xarray as xr
+
+
+def test_landsat_data_preparation(
+    az_url,
+    scale,
+    client_factory,
+    cluster_kwargs={
+        "workspace": "dask-benchmarks-azure",
+        "region": "westeurope",
+        "wait_for_workers": True,
+        "worker_cpu": 2,
+        "worker_memory": "16GiB",
+    },
+    scale_kwargs={
+        "small": {"n_workers": 20},
+        "large": {"n_workers": 150},
+    },
+):
+    with client_factory(
+        **scale_kwargs[scale],
+        env={
+            "AZURE_STORAGE_ACCOUNT_NAME": os.environ["AZURE_STORAGE_ACCOUNT_NAME"],
+            "AZURE_STORAGE_SAS_TOKEN": os.environ["AZURE_STORAGE_SAS_TOKEN"],
+        },
+        **cluster_kwargs,
+    ) as client:  # noqa: F841
+        # Define our STAC endpoint (Microsoft Planetary Computer Catalogue)
+        stac_url = "https://planetarycomputer.microsoft.com/api/stac/v1/"
+        catalog = pystac_client.Client.open(
+            stac_url,
+            modifier=planetary_computer.sign_inplace,
+        )
+
+        if scale == "small":
+            bands = ["cloud_qa", "red", "green", "blue"]
+            box = (
+                10.305842676154782,
+                3.12634712296393502,
+                14.497958439404897,
+                5.052677334304235,
+            )
+        elif scale == "large":
```

Comment:
We haven't really thought too much about scaling in the past. What's the significance of these scaling dimensions? Is it larger an increase data size and more (embarassingly parallel) variables?